src/display/editor/editor.js

```diff
@@ -973,7 +975,8 @@ class AnnotationEditor {
       return this._editToolbar;
     }
     this._editToolbar = new EditorToolbar(this);
-    this.div.append(this._editToolbar.render());
+    this.#editToolbarDiv = this._editToolbar.render();
+    this.div.append(this.#editToolbarDiv);
```

Comment:
Tracking this property manually in this file seems a bit unfortunate, could we instead add a getter in the `EditorToolbar` class, e.g. [placed here](https://github.com/mozilla/pdf.js/blob/0676ea19cf17023ec8c2d6ad69a859c345c01dc1/src/display/editor/toolbar.js#L73), looking something like this?
```javascript
get div() {
  return this.#toolbar;
}
```

That way we could access that via `this._editToolbar?.div` where necessary in this file.